bindings/python/test/coordinate/frame/providers/iers/test_manager.py

```diff
@@ -23,23 +23,28 @@
 from ostk.physics.time import Instant
 from ostk.physics.time import Duration
 
-from ostk.physics.coordinate.frame.providers.iers import Manager
-from ostk.physics.coordinate.frame.providers.iers import BulletinA
-from ostk.physics.coordinate.frame.providers.iers import Finals2000A
+import ostk.physics as physics
+
+################################################################################################################################################################
+
+Manager = physics.coordinate.frame.providers.iers.Manager
+BulletinA = physics.coordinate.frame.providers.iers.BulletinA
+Finals2000A = physics.coordinate.frame.providers.iers.Finals2000A
```

Comment:
Why is this necessary? doesn't importing the whole ostk physics module take more time than just importing the necessary files?

Reply:
Fair, your approach worked too, happy to revert